### PR TITLE
database upgrade fix

### PR DESCRIPTION
**Pull request by sjpadgett**
- add $GLOBALS['ongoing_sql_upgrade'] = true; to status ajax

Fixes a forum issue

#### Short description of what this resolves:
Prior version to 5.0.1 failed because new status polling called globals w/o $GLOBALS['ongoing_sql_upgrade'] = true;

### CODE REVIEW

**Review by bradymiller — APPROVED** [review]

**Comment by bradymiller** [comment]
looks like another good reason to rebuild 6.0.0 packages after next 6.0.0 patch :)

**Comment by sjpadgett** [comment]
Finally have the text templating working pretty good. This is really a neat feature with minimum setup. For instance, the default template context is Encounters so if you've defined an Encounters context with various categories then yields:
image
image

With adding new context and categories the same as Nation Notes:

image

Ending with:
image

Of course my template examples make little clinical sense but, you get the idea..
I think i'll do a video for this or maybe @gutiersa will like to learn by doing on this...

Okay will look over CAMOS then will be done on this!

**Comment by sjpadgett** [comment]
So anyone! If I update CAMOS to use ICD10, will it blow up legacy/history? Is this form still even relevant?  And will someone please , please, tell me how the heck I get myself involved in these issues!!!!

**Comment by gutiersa** [comment]
Big ❤

**Comment by sjpadgett** [comment]
Well then, just for you Sandra:)

File: interface/forms/CAMOS/help.html [hunk 1]
```diff
@@ -163,7 +163,7 @@ <h1>Advanced Use</h1>
 /*billing::CPT4::80054::CMP:: :: ::50.00*/ /*add a cmp for $50*/
 /*billing::CPT4::85024::CBC:: :: ::25.00*/ /*add a cbc*/
 /*billing::CPT4::80061::Lipid Panel:: :: ::50.00*/
-/*billing::ICD9::V70.0::physical:: :: ::0.00*/ /*this is a diagnosis, not a procedure*/
+/*billing::ICD10::V70.0::physical:: :: ::0.00*/ /*this is a diagnosis, not a procedure*/
```
Comment (stephenwaite) [inline interface/forms/CAMOS/help.html:166]: use `Z00.00` instead?

https://www.icd10data.com/Convert/V70.0

Comment (stephenwaite) [reply interface/forms/CAMOS/help.html:166]: thank you @sjpadgett !

Comment (sjpadgett) [reply interface/forms/CAMOS/help.html:166]: cool thanks

Comment (sjpadgett) [reply interface/forms/CAMOS/help.html]: if I knew more about how this form works i'd restyle it. At least that's the excuse i'm going with like, ya know man!
Golden rules to follow: You never say you own a truck, know how to work on cars and most importantly, computers umm, what's that!

**Comment by sjpadgett** [comment]
I think i'll leave this open for a few hours in case I run into something else or someone sees I've missed something.
@bradymiller I escaped the S%&^ out of the diagnosis text!:)

File: interface/forms/CAMOS/help.html (hunk 1 above)
Comment (stephenwaite) [reply interface/forms/CAMOS/help.html]: https://user-images.githubusercontent.com/5618372/107997770-9be06100-6fb1-11eb-9a31-f624e448dfed.mp4

Comment (sjpadgett) [reply interface/forms/CAMOS/help.html]: lol, I know some folks i'd like to drag and drop like that.

**Comment by bradymiller** [comment]
tenor99

**Comment by gutiersa** [comment]
Awe!
*Sandragutierrezmd*
*THINK BEFORE YOU PRINT*
This message, together with any attachments, is intended only for the
addressee. It may contain information that is legally privileged,
confidential, and exempt from disclosure. If you are not the intended
recipient, you are hereby notified that any disclosure, copying,
distribution, use, or any action or reliance on this communication is
strictly prohibited. If you have received this e-mail in error, please
notify the sender immediately by telephone, fax, or return e-mail and
delete the message, along with any attachments.

**Comment by gutiersa** [comment]
@sjpadgett 
So you made a commit above, where can we start adding information regarding your commit, such that when I have some time I can modify the documentation?
What was your commit about?  Looks like a coding fix. This would likely need not be added to the documentation, correct?

**Pull request merged**